Makefile

```diff
@@ -543,6 +565,35 @@ upload-preloaded-images-tar: out/minikube # Upload the preloaded images for olde
 push-storage-provisioner-image: storage-provisioner-image ## Push storage-provisioner docker image using gcloud
 	gcloud docker -- push $(STORAGE_PROVISIONER_IMAGE)
 
+.PHONY: push-docker
+push-docker: # Push docker image base on to IMAGE variable
+	(docker pull $(IMAGE) && (echo "Image already exist"; exit 1) || echo "Image doesn't exist in registry")
+ifndef AUTOPUSH
+	$(call user_confirm, 'Are you sure you want to push $(IMAGE) ?')
+endif
+	docker push $(IMAGE) || gcloud docker -- push $(IMAGE)
+
+.PHONY: push-kic-base-image-gcr
+push-kic-base-image-gcr: kic-base-image ## Push kic-base to gcr
+	$(MAKE) push-docker IMAGE=$(KIC_BASE_IMAGE_GCR)
+
+.PHONY: push-kic-base-image-gh
+push-kic-base-image-gh: kic-base-image ## Push kic-base to github
+	$(MAKE) push-docker IMAGE=$(KIC_BASE_IMAGE_GH)
```

Comment:
please add this line (gh packages need login)
```
docker login docker.pkg.github.com
```
(if its already logged in it will just contnue)